File: kata/7kyu/vowel-count/vowel-count.js
// https://www.codewars.com/kata/54ff3102c1bad923760001f3

const getCount = (str) => {
  let vowelsCount = 0
  const vowels = ['a', 'e', 'i', 'o', 'u']

  vowels.forEach((vowel) => {
    vowelsCount += str.split('').filter((letter) => letter === vowel).length
  })

  return vowelsCount
}

getCount('abracadabra') // -> 5

File: kata/7kyu/vowel-count/vowel-count.py
# https://www.codewars.com/kata/54ff3102c1bad923760001f3

def getCount(sentence):
	vowels = ['a', 'e', 'i', 'o', 'u']
	totalVowels = 0
	for letter in sentence:
		if vowels.__contains__(letter):
			totalVowels += 1
	
	return totalVowels


getCount("aeiou") # -> 5
getCount("y") # -> 0
getCount("bcdfghjklmnpqrstvwxz y") # -> 0
getCount("") # -> 0
getCount("abracadabra") # -> 5